src/x86/sse42.rs

```diff
@@ -255,7 +343,87 @@ pub unsafe fn _mm_cmpestrm(
 }
 
 /// Compare packed strings `a` and `b` with lengths `la` and `lb` using the
-/// control in `imm8`, and return the generated index.
+/// control in `imm8`, and return the generated index. Similar to [`_mm_cmpistri`]
+/// with the excception that [`_mm_cmpistri`] implicityly determines the length of
```

Comment:
I thought I would document what is unique about `_mm_cmpestri` and then add links to `_mm_cmpistri` where I have more examples